certbot/main.py

```diff
@@ -782,6 +782,20 @@ def set_displayer(config):
         displayer = display_util.FileDisplay(sys.stdout)
     zope.component.provideUtility(displayer)
 
+def _post_logging_setup(config, plugins, cli_args):
+    "Perform any setup or configuration tasks that require a logger."
```

Comment:
I thought our plan was to adopt a strict style guide in combination with an autoformatter so that human developers wouldn't spend their time on them ;)